src/App.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Gilda+Display&display=swap');
+
+body {
+    background: url("./assets/background.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    font-family: 'Gilda Display', serif;
+}
+
+.nav-bar {
+    background-color: #248e38;
+    font-size: 36px;
+}
+
+.nav-bar ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
```

Comment:
If you agree with my comment below about factoring out NavBar, we can move these stylings to their own NavBar specific styling file?